test/core/integration/concrete/lockup-linear/streamed-amount-of/streamedAmountOf.t.sol

```diff
@@ -42,7 +37,8 @@ contract StreamedAmountOf_LockupLinear_Integration_Concrete_Test is
         assertEq(actualStreamedAmount, expectedStreamedAmount, "streamedAmount");
     }
 
-    function test_GivenCliffTimeInFuture() external view givenSTREAMINGStatus {
+    function test_GivenCliffTimeInFuture() external givenSTREAMINGStatus {
+        vm.warp({ newTimestamp: defaults.CLIFF_TIME() - 1 });
```

Comment:
well, the test name is: `test_GivenCliffTimeInFuture` 
which means we need to warp before the cliff time, so that the streamed amount is 0

we can indeed remove the warp from the `test_GivenEndTimeInFuture` test

Reply:
Thanks for explaining. I get it now. Previously, `givenSTREAMINGStatus` didn't warping anything inside and so by default the current time was set to the `START_TIME` and therefore it was correct. Now `givenSTREAMINGStatus` warps to 26% which is well beyond cliff time and therefore, you have to warp it to `defaults.CLIFF_TIME() - 1`.

Feel free to mark it resolve then.